Pivot/PivotOrientationView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  PivotOrientationView.swift
+//  Pivot
+//
+//  Created by Devin Frenze on 5/26/17.
+//  Copyright © 2017 Devin Frenze. All rights reserved.
+//
+
+import UIKit
+@IBDesignable
+
+class PivotOrientationView: UIView {
+    override func draw(_ rect: CGRect) {
+        let path = UIBezierPath(ovalIn: rect)
+        UIColor.init(red: 1, green: 0.11764705882353, blue: 0.3843137254902, alpha: 1).setFill()
```

Comment:
oh, also there's color literals now which are sweet, check them out. you can _see the color_